src/main/controller/ExerciseController.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express';
+import db from '../models';
+import { Route } from '../constant/Route';
+
+export class ExerciseController {
+  // (4 - EXP + WP) Retrieve detailed information about an exercise (name, video link, desc, tags)
+  getExerciseDetailed = async (req: Request, res: Response): Promise<any> => {
+    try {
+      if (!req.params.exerciseID) {
```

Comment:
Could you add in sanitization code similiar to what Kim's PR has (we'll merge that soon) here!